languages/python/sqlalchemy-oso/sqlalchemy_oso/auth.py

```diff
@@ -58,6 +60,50 @@ def authorize_model(oso: Oso, actor, action, session: Session, model) -> Query:
         # otherwise
 
     if combined_query is None:
-        return null_query(session)
+        return null_query(session, model)
 
     return combined_query
+
+
+# TODO (dhatch): Api might not be viable.
+def authorize_model_filter(oso: Oso, actor, action, session: Session, model):
```

Comment:
Do we need both this + `authorize_model`?

Reply:
No, I want to drop authorize_model ultimately.  The original reason authorize_model worked over Query was because I thought we may need to add JOINs (which cannot just be constructed as filter expressions).  That turned out not to be necessary, which is why we now have `authorize_model_filter`.

Reply:
Combined this slightly more.

Reply:
510ff36dc3b5eac8c530aa9cf55643321ece987b

Reply:
547a55bf47c17125ecbecbe2ee1fa1406f3d7c6d